[2025-01-29T09:09:08.071+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-29T09:09:08.111+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my-first-python-etl-dag.load manual__2025-01-29T09:08:53.304238+00:00 [queued]>
[2025-01-29T09:09:08.132+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my-first-python-etl-dag.load manual__2025-01-29T09:08:53.304238+00:00 [queued]>
[2025-01-29T09:09:08.134+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-29T09:09:08.178+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load> on 2025-01-29 09:08:53.304238+00:00
[2025-01-29T09:09:08.192+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=508) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-29T09:09:08.195+0000] {standard_task_runner.py:72} INFO - Started process 510 to run task
[2025-01-29T09:09:08.195+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my-first-python-etl-dag', 'load', 'manual__2025-01-29T09:08:53.304238+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpome1sb7a']
[2025-01-29T09:09:08.201+0000] {standard_task_runner.py:105} INFO - Job 17: Subtask load
[2025-01-29T09:09:08.592+0000] {task_command.py:467} INFO - Running <TaskInstance: my-first-python-etl-dag.load manual__2025-01-29T09:08:53.304238+00:00 [running]> on host 0d287206d23f
[2025-01-29T09:09:08.695+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='olalekanrasaq1331@gmail.com' AIRFLOW_CTX_DAG_OWNER='Akinkunmi' AIRFLOW_CTX_DAG_ID='my-first-python-etl-dag' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2025-01-29T09:08:53.304238+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-29T09:08:53.304238+00:00'
[2025-01-29T09:09:08.697+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-29T09:09:08.697+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-29T09:09:08.698+0000] {logging_mixin.py:190} INFO - Current task name:load state:running start_date:2025-01-29 09:09:08.113511+00:00
[2025-01-29T09:09:08.699+0000] {logging_mixin.py:190} INFO - Dag name:my-first-python-etl-dag and current dag run status:running
[2025-01-29T09:09:08.700+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-29T09:09:08.701+0000] {logging_mixin.py:190} INFO - Inside Load
[2025-01-29T09:09:08.701+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-29T09:09:08.713+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-29T09:09:08.713+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my-first-python-etl-dag, task_id=load, run_id=manual__2025-01-29T09:08:53.304238+00:00, execution_date=20250129T090853, start_date=20250129T090908, end_date=20250129T090908
[2025-01-29T09:09:08.726+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-29T09:09:08.727+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-29T09:09:08.728+0000] {logging_mixin.py:190} INFO - Dag name:my-first-python-etl-dag queued_at:2025-01-29 09:08:53.318988+00:00
[2025-01-29T09:09:08.728+0000] {logging_mixin.py:190} INFO - Task hostname:0d287206d23f operator:PythonOperator
[2025-01-29T09:09:08.780+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-29T09:09:08.973+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-29T09:09:08.977+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
